src/frontend/src/pages/PPQPage.tsx

```diff
@@ -19,36 +19,36 @@ function PPQLandingPage() {
 
       <div className="  ppq-section">
         <div className="row what-is-a-ppq">
-        <div className='col-md-6'>
-          <h2 >PIA Pathway Questionnaire</h2>
-          <br />
-          <p className='col-mb-6'>
-            The PIA Pathways Questionnaire asks for some basic information about
-            the initiative you're assessing in the PIA. Your answers will help
-            us estimate{' '}
-          </p>
-         <ul className='col-mb-6 list'>
-          <li>The relative complexity of your PIA </li>
-          <li> Which PIA template you should fill out</li>
-          <li>Where to go for help with your PIA</li>
-          </ul> 
-          <div data-cy="ppq-btn" className="ctas">
-            <Link to="/ppq-form" className="btn-primary">
-              Get started{' '}
-              <FontAwesomeIcon className="icon" icon={faChevronRight} />
-            </Link>
+          <div className="col-md-6">
+            <h2>PIA Pathway Questionnaire</h2>
+            <br />
+            <p className="col-mb-6">
+              The PIA Pathways Questionnaire asks for some basic information
+              about the initiative you're assessing in the PIA. Your answers
+              will help us estimate{' '}
+            </p>
+            <ul className="col-mb-6 list">
+              <li>The relative complexity of your PIA </li>
+              <li> Which PIA template you should fill out</li>
+              <li>Where to go for help with your PIA</li>
+            </ul>
+            <div data-cy="ppq-btn" className="ctas">
+              <Link to="/ppq-form" className="btn-primary">
+                Get started{' '}
```

Comment:
Is this hardcoding a space into the html? Is this best practice or something that we can handle in css?

@Bottle7

Reply:
Typically, we space things using styling and not DOM elements.

Reply:
Lets make the change then

Reply:
100% agreement to this point. I just commented the same thing above.